### PR TITLE
Release 3.1.0

### PR DESCRIPTION
**Pull request by connectdotz**
prepare release 3.1.0:

- update version
- update CHANGELOG
- update vscode manifest
  - `galleryBanner`:  to correct marketplace banner background color appeared to be close to the rating stars' color.
  - `extensionKind`:  vscode changed the type from string to array.
- update outdated README and add releases link

(A quick look, at least for the README, would be appreciated)

### CODE REVIEW

**Comment by connectdotz** [comment]
@stephtr sorry to ping you again, I was trying to finish 3.1 before the holiday

File: README.md [hunk 1]
```diff
@@ -44,21 +44,23 @@ Orta Therox ([@orta](https://github.com/orta)), Sean Poulter ([@seanpoulter](htt
...
 
-If you have a more complex setup, it can probably be supported, check out the settings. They are all prefixed with `jest`.
-
-Also, you should use Jest 17+, however 16 works - it will just offer a warning. We're aiming to try and do current Jest version - 1, but won't specifically drop support for older versions unless we're forced into it.
+Out of the box, this extension should work for most simple/standard jest and react projects. However, if you have a more sophisticated project or custom jest command, it most likely won't be sufficient but you can easily customize it with various settings, such as `jest.pathToJest` where you can specify how you usually run your jest tests. These settings are all prefixed with `jest`.
 
+If you encountered any difficulty or have suggestions, please do not hesitate to [ask](https://github.com/jest-community/vscode-jest/issues), we have an active community that you might find helpful. 
 
 ## How to start the Jest?
```
Comment (stephtr) [inline README.md]: Maybe let's replace it with "How to start Jest?"

File: README.md [hunk 2]
```diff
@@ -44,21 +44,23 @@ Orta Therox ([@orta](https://github.com/orta)), Sean Poulter ([@seanpoulter](htt
...
 * Adds syntax highlighting to snapshot files.
 * A one button update for failed snapshots.
-* Show coverage information in files being tested. (_requires coverage to be collected by your jest config_)
+* Show coverage information in files being tested. 
+* Help debug jest tests in vscode.
 
 ## How to get it?
 
 Open up VS Code, go search for the extension "Jest".  
 As an alternative, install with 1-click through [Visual Studio Marketplace - Orta.vscode-jest](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest).
```
Comment (stephtr) [inline README.md]: Can we maybe change that to something like:
> Simply open [Jest - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) and click "Install".
> Alternatively open Visual Studio Code, go to the extension view and search for "Jest".

File: README.md [hunk 3]
```diff
@@ -44,21 +44,23 @@ Orta Therox ([@orta](https://github.com/orta)), Sean Poulter ([@seanpoulter](htt
...
+ 
+The detailed releases and migration help: see [releases](https://github.com/jest-community/vscode-jest/releases).
 
 ## How to get it set up?
 
-This project has the expectation that you would run something like `npm run test` which _just_ looks like `jest` in the `package.json`. So, please keep your configuration inside the `package.json` as opposed to using command line arguments.
```
Comment (stephtr) [inline README.md]: What about "This extension runs on top of your Jest installation. When being started, it has the expectation that the Jest environment is properly set up, i.e. that the tests can be executed in VS Code's shell.

File: README.md [hunk 4]
```diff
@@ -44,21 +44,23 @@ Orta Therox ([@orta](https://github.com/orta)), Sean Poulter ([@seanpoulter](htt
...
+This project deploys a thin layer to simulate how users run their jest tests. It has the expectation that, upon launch, users already have their jest environment properly setup, i.e. they can execute the basic jest tests in vscode shell. 
 
-If you have a more complex setup, it can probably be supported, check out the settings. They are all prefixed with `jest`.
-
-Also, you should use Jest 17+, however 16 works - it will just offer a warning. We're aiming to try and do current Jest version - 1, but won't specifically drop support for older versions unless we're forced into it.
+Out of the box, this extension should work for most simple/standard jest and react projects. However, if you have a more sophisticated project or custom jest command, it most likely won't be sufficient but you can easily customize it with various settings, such as `jest.pathToJest` where you can specify how you usually run your jest tests. These settings are all prefixed with `jest`.
```
Comment (stephtr) [inline README.md]: For the sake of reading flow, I would separate it into two different sentences: "...it most likely won't be sufficient. However, you can easily customize it..."
"These settings are all prefixed..." -> "All of this extension's settings are prefixed..."

**Comment by connectdotz** [comment]
thanks @stephtr, I incorporated some of your comments. On to the release...

**Pull request merged**